### DISCUSSION TITLE
Download fails for website that worked before

### DISCUSSION THREAD

**Discussion by xezoxeon** [aclap-dev/video-downloadhelper#1354] [Help and Q&A]
### Your question

Why is it that I am presented with the error "Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!)" when I try to download... it worked before.

### Addon & CoApp version and details

version: 9.1.0.44
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"a1c80bf5b445417c"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "http_media_download_strategy": "inbrowser",
    "used_history_button": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "F:\\Movies"
        }
    },
    "theme": "dark",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "default_action": "download_as",
    "use_wide_ui": true,
    "open_count_store": 4,
    "record_download_history": true
}

**Comment by Mattmjs** [comment]
Websites can change things at any point in time.
If you can share a page where it is happening it
will help. If it's a private site that requires a login 
then follow these steps.

1. Click the 3 dot button next to download.
2. Look down to details and click on it.
3. A page opens with more information.
4.  **Ctrl + A** to select all on page.
5.  **Ctrl + C** to copy information.
6.  **Ctrl + V** to paste the information here.

**Reply by xezoxeon** [reply]
https://soaper.tv/

**Reply by Mattmjs** [reply]
Ok it does fail, but when I use ffmpeg directly which is what VDH uses it proceeds to download it. [Example video](https://soaper.tv/movie_9dG7YqND05.html)

I don't know why it would fail, it looks like it should work. I think the dev can offer fix for it. @paulrouget
